08-rails/active-record-migrations.md

```diff
@@ -91,6 +91,21 @@ end
 
 Once we have the migration created, we must again run `rails db:migrate` for the migration to take effect.
 
+## Practice
+
+In our sample Rails project perform the following actions:
+
+1.  Create a new model named: `Author`, an author should only have a `name` field (String).  Remember to run your migration with `rails db:migrate`
+2.  Then add a migration to add a `publisher` field to the model.
+
+Then run `rails db:migrate:status` to verify that the migrations have run successfully.
+
+You can then destroy your `Author` model with:
+
+```bash
+rails destroy model Author name:string publisher:string
```

Comment:
I will pretty much never have them do this command, so i'm wondering if we should just avoid adding the author altogether. They should run exactly the model and migration commands in task list (i think?)